libcontainer/devices/devices_test.go

```diff
@@ -61,3 +62,33 @@ func TestHostDevicesIoutilReadDirDeepFailure(t *testing.T) {
 		t.Fatalf("Unexpected error %v, expected %v", err, testError)
 	}
 }
+
+func HostDevicesAllValid(t *testing.T) {
```

Comment:
The function name needs to have `Test` prefix

Reply:
Oops.